src/test/javascript/spec/component/competencies/competency-management/competency-management.component.spec.ts

```diff
@@ -60,6 +65,18 @@ describe('CompetencyManagementComponent', () => {
                 provideRouter([]),
                 MockProvider(AccountService),
                 MockProvider(AlertService),
+                {
+                    provide: AlertService,
+                    useClass: MockAlertService,
+                },
+                {
+                    provide: ProfileService,
+                    useClass: MockProfileService,
+                },
+                {
+                    provide: AlertService,
+                    useClass: MockAlertService,
+                },
```

Comment:
```suggestion
                {
                    provide: AlertService,
                    useClass: MockAlertService,
                },
                {
                    provide: ProfileService,
                    useClass: MockProfileService,
                },
```
One should be enough